apache-hbase-mixin/signals/cluster.libsonnet

```diff
@@ -0,0 +1,316 @@
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'max without (clusterid,deadregionservers,liveregionservers,servername,zookeeperquorum,isactivemaster) (server_num_region_servers{%(queriesSelector)s, isactivemaster="true"} * 0  + 1 )',
+          },
+        },
+      },
+
+      nonMasterStatusHistoryNumRegionServers: {
+        name: 'Non-master status history number of region servers',
+        type: 'gauge',
+        description: 'The number of region servers for the non-master status history.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: '(max without (clusterid,deadregionservers,liveregionservers,servername,zookeeperquorum,isactivemaster) (server_num_region_servers{%(queriesSelector)s, isactivemaster="false"}) * 0)',
+          },
+        },
+      },
+
+      // Master and cluster status
+      liveRegionServers: {
+        name: 'Live RegionServers',
+        type: 'gauge',
+        description: 'Number of RegionServers that are currently live.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'server_num_region_servers{%(queriesSelector)s, isactivemaster="true"}',
+          },
+        },
+      },
+
+      deadRegionServers: {
+        name: 'Dead RegionServers',
+        type: 'gauge',
+        description: 'Number of RegionServers that are currently dead.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'server_num_dead_region_servers{%(queriesSelector)s, isactivemaster="true"}',
+          },
+        },
+      },
+
+      // Connections
+      masterConnections: {
+        name: 'Master connections',
+        type: 'gauge',
+        description: 'Number of open connections to the master.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters',
+          },
+        },
+      },
+
+      regionServerConnections: {
+        name: 'RegionServer connections',
+        type: 'gauge',
+        description: 'Number of open connections to RegionServers.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - RegionServers',
+          },
+        },
+      },
+
+      // Authentication
+      masterAuthenticationSuccess: {
+        name: 'Master authentication successes',
+        type: 'raw',
+        description: 'Rate of successful authentications to the master.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_authentication_successes{%(queriesSelector)s})',
```

Comment:
Example: This signal reads as "Rate of ..." but the query doesn't use the `rate(...)` function, which most likely will lead them to be displayed as as raw counter values instead of rates.